packages/api/test/e2e/staking.e2e.ts

```diff
@@ -207,8 +207,7 @@ describe('Staking Operations', () => {
   });
 
   test('Calculate accruedPayout via RPC', async done => {
-    const rewardDestinationAddress = '5FEe8Ht1ZTzNjQcvrxbLxnykA2EXfqN5LMog2gaNPus4tfZR';
-    const accruedPayout = await api.rpc.staking.accruedPayout(rewardDestinationAddress);
+    const accruedPayout = await api.rpc.staking.accruedPayout(stash.address);
     expect(accruedPayout.toNumber()).toBeGreaterThanOrEqual(0);
     done();
```

Comment:
there will always be a reward now
```suggestion
    expect(accruedPayout.toNumber()).toBeGreaterThan(0);
    done();
```

Reply:
done..